src/Contao/SessionStorage.php

```diff
@@ -94,7 +128,8 @@ public function has($name)
     public function get($name)
     {
         $this->load();
-        return isset($this->attributes[$name]) ? $this->attributes[$name] : null;
+
+        return $this->attributes[$name];
```

Comment:
This will trigger a notice if not set, we should not introduce notices!

Reply:
I have rework this

Reply:
93afa387